keps/sig-ui/NNNN-kep-template/README.md

```diff
@@ -0,0 +1,810 @@
+
+### Background
+
+Headlamp is a Kubernetes UI that can run as a Desktop application and an in-cluster web interface. It is designed to be extended using its robust plugin system. This allows vendors or projects to adapt it to their use case and for the project to deliver builds for various use cases.
+
+Headlamp is currently available in two versions.
+* **Headlamp** refers to the project and the default distribution of Headlamp which ships with core Kubernetes UI functionality (see the next bullet point) plus a set of plugins that the project thinks are generally useful for all users in order to provide a rich and relevant Kubernetes user experience.
+* **Headlamp Base** does not include any plugins. It only includes features that the project deems universally desired in a Kubernetes UI. This is designed for builders looking to have a solid base on which to extend the user experience. They would generally create custom plugins to customize the user experience, include any existing plugins they desire and then theme the UI to fit their branding.
+
+Each version is available as a desktop experience and an in-cluster experience. For the default build, the set of plugins differs between the in-cluster and desktop installations as there are different user profiles for each environment. For example, the plugin that allows managing plugins is not included in the in-cluster environment as the cluster admin generally determines which plugins to provide to the cluster users.
+
+### Existing Dashboard project
+The Kubernetes project already has a Dashboard project. However, we think that the Headlamp project has a number of advantages that make it a better solution to build on going forward. Some of these are:
+* Has a robust plugin system for customization and integrations
+* Supports both desktop and in-cluster mode
+* Is under very active development and has dedicated resources assigned to it in addition to the active external contributors
+* Is built on a very common technology stack (Typescript, React, Material UI, Vite, Go)
+* Has more advanced features (multi-cluster, graph view, powerful plugin system, 3rd party plugins)
+* Addresses many more use cases (see below)
+
+
+### Goals
+
+With this proposal we aim to provide the Kubernetes project with the following.
+* A tool to more easily onboard new and beginner-level users
+* A UI that serves as a base for vendors to build upon
+* A more modern UI developer environment
+* A more feature-rich UI
+* A desktop user experience
+* An integration point for other CNCF and external projects, especially those without a UI focus
+
+
+### Non-Goals
+
+This proposal does not propose to force people who are using the current Dashboard to move to Headlamp.
```

Comment:
Use this section too. Try to think "defensive" on things that might be asked and you don't want to tackle as part of the initial KEP. Think of people asking for this later on, and you can point out "that was stated as a non-goal" of the KEP that is approved.

 I don't know about the KEP and what is reasonable to do or not, but I can imagine a non-goal might be to not support automatic migration of the current k8s dashboard, for example.

Try to really use this section to narrow the scope of the KEP, to clarify some of the goals even (like "we say X is a goal, but Y is not part of the goal")